File: fizzBang.js
// FIZZBANG for an integer

function fizzBang(n) {
    if (n % 3 === 0 && n % 5 === 0) {
        console.log('FIZZBANG')
    } else if (n % 3 === 0) {
        console.log('FIZZ')
    } else if (n % 5 === 0) {
        console.log('BANG')
    } else {
        console.log('There is neither FIZZ nor BANG')
    };
};


fizzBang(7)

// FIZZBANG for a range

function fizzBangRng(m, n) {
    for (i = m; i >= m && i <= n; i++) {
        if (i % 3 === 0 && i % 5 === 0) {
            console.log('FIZZBANG')
        } else if (i % 3 === 0) {
            console.log('FIZZ')
        } else if (i % 5 === 0) {
            console.log('BANG')
        } else {
            console.log(i)
        };
    };
};

fizzBangRng(1, 16)

File: fizz_bang.py
# FIZZ BANG #
# create an function such that an integer divisible
# by 3 iwill retrun 'FIZZ', if divisible by 5
# it will retrun 'BANG' and if divisible by both 3 and 5
# 'FIZZBANG' will be returned.

# FIZZ BANG FOR AN INTEGER
def fizz_bang(n):
    if n % 3 == 0 and n % 5 == 0:
        return 'FIZZBANG'
    if n % 3 == 0:
        return 'FIZZ'
    if n % 5 == 0:
        return 'BANG'

# FIZZ BANG FOR A RANGE


def fizz_bang_rng(rng):
    for n in rng:
        if n % 3 == 0 and n % 5 == 0:
            print('FIZZBANG')
        elif n % 3 == 0:
            print('FIZZ')
        elif n % 5 == 0:
            print('BANG')
        else:
            print(n)


fizz_bang_rng(range(1, 16))
